scorep/__main__.py

```diff
@@ -19,17 +20,22 @@ def scorep_main(argv=None):
     parse_scorep_commands = True
 
     keep_files = False
+    verbose = False
     no_default_threads = False
     no_default_compiler = False
     no_instrumenter = False
     instrumenter_type = "profile"
 
     for elem in argv[1:]:
         if parse_scorep_commands:
-            if elem == "--mpi":
+            if elem == "--":
+                parse_scorep_commands = False
```

Comment:
Hmmm ... why did you do this explicit?

Reply:
This is to allow the usual `--` syntax to stop argument parsing for commands running commands. E.g.: `srun --ntasks=1 -- foobar --help`
There might be users used to that and it is a 2-line addition, so why not?

Reply:
alright. I thought about it. I do agree.